plumbing/format/gitignore/dir.go

```diff
@@ -0,0 +1,50 @@
+package gitignore
+
+import (
+	"io/ioutil"
+	"strings"
+
+	"gopkg.in/src-d/go-billy.v2"
```

Comment:
As you merged the request, I understand you have already updated this? Is there anything left for me to do?